Assets/StreamingAssets/Quests/_TUTOR__.txt

```diff
@@ -8,7 +8,8 @@ Quest: _TUTOR__
 -- Message panels
 QRC:
 
-RumorsDuringQuest:  [1005]
+
+RumorsPostfailure:  [1006]
```

Comment:
Do these only occur if you say No to the tutorial, or does completing the tutorial to the end count as failure? I guess I'll test it out

Reply:
Oh just saw that the last page just calls the _no_ task, got it

Reply:
The way this is written, all ends lead to failure. The only way to get a quest to be a success is if you do "give pc ITEM" of some kind, or "give pc nothing." They invoke message 1004 as well. Anything without them is a failure.